### PR TITLE
Update DevOps Rules

### PR DESCRIPTION
**Pull request by AndreasLengkeek**

### CODE REVIEW

File: rules/are-you-flexible-with-the-order-you-do-the-work-i-e-cancelling-a-sprint/rule.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 type: rule
 archivedreason: 
-title: Are you flexible with the order you do the work (i.e. Cancelling a sprint)?
+title: Do you have flexibility with the order you do the work (i.e. cancelling a Sprint)?
```
Comment (pierssinclairssw) [inline rules/are-you-flexible-with-the-order-you-do-the-work-i-e-cancelling-a-sprint/rule.md:4]: Feels more confusing than before

File: rules/things-to-automate-stage-2/rule.md [hunk 2]
```diff
@@ -17,10 +17,12 @@ redirects:
 
 ---
 
-Once you’ve identified the manual processes in [Stage 1](/things-to-measure), you can start looking at automation. The best tool for build and release automation is Azure DevOps.
+Once you’ve identified the manual processes in [Stage 1](/things-to-measure), you can start looking at automation. The two best tools for build and release automation are Github (public projects) and Azure DevOps (private projects).
```
Comment (pierssinclairssw) [inline rules/things-to-automate-stage-2/rule.md:20]: Are we advocating against GitHub Private Projects? How come?

Comment (AndreasLengkeek) [reply rules/things-to-automate-stage-2/rule.md:20]: I wanted to give a recommendation for why to use one or the other. At this point GitHub has better public/open source projects so DevOps got the other one

File: rules/are-you-flexible-with-the-order-you-do-the-work-i-e-cancelling-a-sprint/rule.md (hunk 1 above)
Comment (AndreasLengkeek) [reply rules/are-you-flexible-with-the-order-you-do-the-work-i-e-cancelling-a-sprint/rule.md]: Removed

**Review by pierssinclairssw — APPROVED** [review]

**Pull request merged**